### ISSUE TITLE
AA standards

### ISSUE THREAD

**Issue by dtedesco1** [dtedesco1/nextjs-markdown-boilerplate#2]
[belmead on HN](https://news.ycombinator.com/reply?id=45108044&goto=item%3Fid%3D45071592%2345108044) points out: "the color contrast for copy and other elements doesn't meet AA accessibility standards"

Plz address
